### PR TITLE
Declare empty slice

### PR DESCRIPTION
**Pull request by rstorr**

### CODE REVIEW

File: tenets/codelingo/go/declare_empty_slice/codelingo.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+      @review comment
+      go.assign_stmt(depth = any):
+        go.rhs:
+          go.composite_lit:
+            go.array_type:
+              go.ident
+
+              exclude:
+                go.basic_lit
```
Comment (BlakeMScurr) [inline tenets/codelingo/go/declare_empty_slice/codelingo.yaml]: Do you need this? Shouldn't excluding `go.elts` be enough? Even a slice with a single element has a `go.elts` fact.

File: tenets/codelingo/go/declare_empty_slice/example.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  var t []string
+  t := []string{}
+  t := [2]string{2,3}
+  
+  i := 0
+  for i < 10 {
+    var w []string
+    w := []string{}
+    w := [2]string{2,3}
```
Comment (BlakeMScurr) [inline tenets/codelingo/go/declare_empty_slice/example.go]: I don't think this will compile, but that's a minor issue - it does parse.

Comment (rstorr) [reply tenets/codelingo/go/declare_empty_slice/example.go]: which line? the whole loop?

Comment (waigani) [reply tenets/codelingo/go/declare_empty_slice/example.go]: yep, needs to be type `int`

File: tenets/codelingo/go/declare_empty_slice/codelingo.yaml (hunk 1 above)
Comment (rstorr) [reply tenets/codelingo/go/declare_empty_slice/codelingo.yaml]: You're right I'll take it out.

**Pull request closed without merging**